examples/quickstart-pytorch/client.py

```diff
@@ -107,7 +107,7 @@ def apply_transforms(batch):
 
 # Load model and data (simple CNN, CIFAR-10)
 net = Net().to(DEVICE)
-trainloader, testloader = load_data(partition_id=partition_id)
+trainloader, testloader = load_data(partition_id=0)
```

Comment:
```suggestion
trainloader, testloader = load_data(partition_id=partition_id)
```

I think this shouldn't be changed right?